### ISSUE TITLE
Telemetry should point out the target URL[s] for metrics

### ISSUE THREAD

**Issue by benshayden** [catapult-project/catapult#2820]
https://codereview.chromium.org/2323533003/diff/200001/tracing/tracing/model/helpers/chrome_model_helper.html

Telemetry should add some metadata to the trace so that metrics can conclusively determine the target renderer instead of using this heuristic.
Is it the story URL that identifies the target renderer?
Should we use the FrameBlameContexts to match up the story URL with the renderer process?

@nedn @deepanjanroy @eakuefner

**Comment by nedn** [comment]
Not really. I believe the direction to move forward on this is to collect relevant metrics on all renderer & add them to the histogram.

This is for the cases of:
1) Page navigate to different domains
2) Multi-tabs test

@nyaxt is planning on modifying that code so that we collect FMP metrics on multiple renderers and not just one.
